kubernetes-manifests/contacts.yaml

```diff
@@ -28,7 +28,7 @@ spec:
       terminationGracePeriodSeconds: 5
       containers:
       - name: contacts
-        image: gcr.io/bank-of-anthos/contacts:latest
+        image: gcr.io/sunnyqi/contacts:0616201501
```

Comment:
Seems like this should stay bank-of-anthos?

Reply:
oh right! didn't mean to commit that